extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -1,13 +1,17 @@
 /*global define*/
 /*jshint maxlen:125, camelcase:false, maxdepth:7*/
 define('ext.wikia.adEngine.provider.gpt.googleTag', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adTracker',
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (doc, log, window) {
+], function (adContext, adTracker, doc, log, window) {
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.provider.gpt.googleTag',
+		sourcePointClientId = 'rMbenHBwnMyAMhR',
```

Comment:
I wonder if we could separate Source Point and GPT more. So, the GPT code doesn't include Source Point properties. Maybe we could provide an optional `sourcePointTag` module and once it's provided change stuff in `init`? Or just have it child of GoogleTag and just overwrite `init()`? I'm just "thinking aloud". We can talk about it offline ;)